tests/spec/core/idl-index-spec.js

```diff
@@ -264,6 +267,7 @@ dictionary PromptResponseObject {
         <h2>PASS</h2>
         <p>Custom paragraph.</p>
       </section>
+      <section id="conformance"></section>
```

Comment:
Multiple elements with the same ID are a weird situation, what's this for?

Reply:
this is ok I think... It's part of a different test, but above it looks like the same test.

Reply:
(note the above is on line 203, and this is line 268)

Reply:
Oops, my mistake, sorry.